packages/clima_ui/lib/screens/settings_screen.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:clima_ui/screens/about_screen.dart';
+import 'package:clima_ui/widgets/reusable_widgets.dart';
+import 'package:clima_ui/widgets/unit_dialog.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+
+import 'appearance_screen.dart';
+
+class SettingScreen extends HookWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      resizeToAvoidBottomInset: false,
+      appBar: AppBar(
+        leading: IconButton(
+            icon: const Icon(Icons.arrow_back),
+            onPressed: () {
+              Navigator.of(context).pop();
+            }),
+        title: Text(
+          'Settings',
+          style: Theme.of(context).appBarTheme.textTheme.subtitle1,
+        ),
+      ),
+      body: SingleChildScrollView(
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          children: <Widget>[
+            const SettingsHeader(
+              title: 'General',
+            ),
+            SettingsTile(
+              title: 'Appearance',
+              subtitle: 'Customization of appearance',
```

Comment:
I don't think this subtitle is necessary; it doesn't really add anything to the title.